src/services/usersService.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const {
+  getNewId,
+} = require('../helper');
+
+let users = [];
+
+const setInitialUsers = () => {
+  users = [];
+};
+
+class UserService {
+  getAll() {
+    return users;
+  };
+  getOne(userId) {
+    return users.find(({ id }) => id === +userId) || null;
+  };
+  create(name) {
```

Comment:
Let's add some spacing between methods

```suggestion
  getAll() {
    return users;
  };
  
  getOne(userId) {
    return users.find(({ id }) => id === +userId) || null;
  };
  
  create(name) {
```